src/vmm/src/vstate/system.rs

```diff
@@ -11,7 +11,7 @@ use std::{
 };
 
 use kvm_bindings::KVM_API_VERSION;
-use kvm_ioctls::Kvm;
+use kvm_ioctls::{Error as kvm_ioctls_err, Kvm};
```

Comment:
The Rust convention is to have data types in CamelCase, like: `KvmIoctlsError`

Reply:
Good to know thanks.